src/process.c

```diff
@@ -463,59 +466,450 @@ cc_run_hook(struct oci_cfg_hook* hook, const gchar* state,
 	return result;
 }
 
+
+/*!
+ * Obtain the network configuration by querying the network namespace.
+ *
+ * \param[in,out] config \ref cc_oci_config.
+ *
+ * \return \c true on success, else \c false.
+ */
+static gboolean
+cc_oci_vm_netcfg_get (struct cc_oci_config *config)
+{
+	if ((! config)) {
+		return false;
+	}
+
+	/* TODO: We need to support multiple networks */
+	if ( !cc_oci_network_discover ("eth0", config)) {
```

Comment:
@mcastelino Maybe you could comment on how this works with a hardcoded `eth0` string ?